### PR TITLE
feat: [CCM-7144]: Azure connector UI changes

### PR DESCRIPTION
**Pull request by jyotiarora-harness**
##### Summary:

CE Azure connector UI changes

##### Jira Links:

https://harness.atlassian.net/browse/CCM-7144

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Comment by jyotiarora-harness** [comment]
trigger cypress

File: src/modules/35-connectors/common/ConnectorCreationInstructionList/ConnectorCreationInstructionList.module.scss [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
 .instructionList {
   line-height: 40px;
+  color: var(--grey-800);
+  font-size: 13px;
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/common/ConnectorCreationInstructionList/ConnectorCreationInstructionList.module.scss:17]: use FontVariation

Comment (jyotiarora-harness) [reply src/modules/35-connectors/common/ConnectorCreationInstructionList/ConnectorCreationInstructionList.module.scss:17]: This class is used on li element not on text element, that's why used like this

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/Steps/Billing/AzureExistingBillingExports.tsx [hunk 2]
```diff
@@ -17,39 +18,61 @@ interface Props {
...
-    </Container>
+    <>
+      <Text color={Color.GREY_800} font={{ variation: FontVariation.BODY }} padding={{ bottom: 'small' }}>
+        {getString('connectors.ceAzure.existingExports.instruction', {
+          reportCount: props.existingBillingExports.length
+        })}
+      </Text>
+      <Container className={css.existingBeTable}>
+        <ExpandingSearchInput
+          onChange={/* istanbul ignore next */ text => onChange(text.trim())}
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/Steps/Billing/AzureExistingBillingExports.tsx:40]: why ignoring this line?

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss [hunk 3]
```diff
@@ -146,13 +151,42 @@
...
+    padding: var(--spacing-xlarge);
+
+    ul {
+      padding: var(--spacing-medium);
+      margin: 0;
+    }
+  }
+
+  .hintsLineItem {
+    padding-bottom: 12px;
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:174]: use var

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss [hunk 4]
```diff
@@ -146,13 +151,42 @@
...
+    font-size: 13px;
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:175]: use font variation

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss [hunk 5]
```diff
@@ -146,13 +151,42 @@
...
+    line-height: 30px;
+    color: var(--grey-800);
+  }
+
+  .infoCard {
+    background: var(--blue-50);
+    padding: 10px var(--spacing-medium);
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:182]: use var

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss [hunk 6]
```diff
@@ -72,7 +71,7 @@
   }
 
   .header {
-    color: #22222a !important;
+    color: var(--grey-800) !important;
     font-weight: 600 !important;
     font-size: 20px !important;
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:76]: SAA

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector.module.scss [hunk 7]
```diff
@@ -25,7 +25,7 @@
   }
 
   .header {
-    color: #22222a !important;
+    color: var(--grey-800) !important;
     font-weight: 600 !important;
     font-size: 20px !important;
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector.module.scss:30]: SAA

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss (hunk 4 above)
Comment (jyotiarora-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:175]: This class is used on li element not on text element, that's why used like this

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss (hunk 5 above)
Comment (jyotiarora-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:182]: There is no spacing defined for 10px in guidelines

Comment (meenakshi-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:182]: We should use something nearest to 10px. Mocks will have different values.

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss (hunk 3 above)
Comment (meenakshi-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:174]: Same as below

File: src/modules/35-connectors/common/ConnectorCreationInstructionList/ConnectorCreationInstructionList.module.scss (hunk 1 above)
Comment (jyotiarora-harness) [reply src/modules/35-connectors/common/ConnectorCreationInstructionList/ConnectorCreationInstructionList.module.scss:17]: As discussed we need this on multiple places, so we should add this in uicore, will initiate a discussion with design team for this

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss (hunk 6 above)
Comment (jyotiarora-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector_new.module.scss:76]: Existing code changes, taking up in follow up PR
As changes are impacting other components as well, so need to test thoroughly

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector.module.scss (hunk 7 above)
Comment (jyotiarora-harness) [reply src/modules/35-connectors/components/CreateConnector/CEAzureConnector/CreateCeAzureConnector.module.scss:30]: Existing code changes, taking up in follow up PR
As changes are impacting other components as well, so need to test thoroughly

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Comment by jyotiarora-harness** [comment]
trigger cypress

File: src/modules/35-connectors/components/CreateConnector/CEAzureConnector/__tests__/CEAzureConnector.test.tsx [hunk 8]
```diff
@@ -164,8 +164,20 @@ describe('Create Azure connector Wizard', () => {
...
       clickSubmit(container)
     })
-
     expect(getByText('connectors.ceAzure.existingExports.instruction')).toBeDefined()
+    const searchButton = container.querySelector('[class*="ExpandingSearchInput"]')
+    expect(searchButton).toBeDefined()
+    act(() => {
+      fireEvent.click(searchButton!)
+    })
+
```
Comment (meenakshi-harness) [inline src/modules/35-connectors/components/CreateConnector/CEAzureConnector/__tests__/CEAzureConnector.test.tsx:173]: after act we should wait for some element else this test could be flaky

**Review by meenakshi-harness — APPROVED** [review]

**Comment by meenakshi-harness** [comment]
@jyotiarora-harness As discussed, please take unresolved comments in follow up PR

**Review by rishik30 — APPROVED** [review]

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Comment by jyotiarora-harness** [comment]
trigger cypress

**Pull request merged**